server/wfnews-api-rest-endpoints/src/main/java/ca/bc/gov/nrs/wfnews/api/rest/v1/endpoints/PublicPublishedIncidentEndpoint.java

```diff
@@ -37,6 +37,8 @@ public Response getPublishedIncidentList(
 		@ApiParam("Filter on fires that are new within 24 hours") @QueryParam("newFires") Boolean newFires,
 		@ApiParam("Filter on fire centre code") @QueryParam("fireCentreCode") String fireCentreCode,
 		@ApiParam("Filter on fire centre name") @QueryParam("fireCentreName") String fireCentreName,
+		@ApiParam("Filter on discovery date") @QueryParam("discoveryDate") String discoveryDate,
```

Comment:
The query params "discoveryDate" and "createDate" do not make it clear that its actually doing a "greater than check". 

- Please rename the attributes so they identify what the param is actually for, or...
-  optionally provide "from/to" dates for discovery and create date... or...
- Make it an equals, not a greater than

Reply:
I will address this